### ISSUE TITLE
2 Quick Edits: y-axis clarification, searching datasets/materials

### ISSUE THREAD

**Issue by beccalau** [HybriD3-database/MatD3#15]
Hello,

As more datasets have been submitted into Hybrid3, @volkerblum , @andrewjlevin and I have discussed a couple of minor tweaks to the database that could be helpful as the database continues to grow. @raullaasner We were hoping you could take a quick look at the following points and make these edits? Hopefully these edits are rather minor.

1. When adding a figure/graph to a dataset, there seems to be a lack of clarity when labeling the y-axis. As of now, there is a label next to "Primary Property" that says "y-axis". When labeling the axis, a few contributors were unclear as to whether the primary property is the title of the y-axis, or the title of the graph. The database registers the entry of the "Primary Property" as the title, and only registers as the y-axis label as well when no other entry is in the "Primary Property Label" field below. Is it possible to clarify this? Perhaps putting "(y-axis)" next to the "Primary Property Label" instead of next to the "Primary Property" field? 

2. Would it be possible to add two drop down fields when searching for terms? Specifically, we would like to add "Dataset number" and "Material Number" or something along those lines. 

Thank you so much! Also, @volkerblum and @andrewjlevin, if you have any more points to add to this, please do so.

**Comment by raullaasner** [comment]
The primary or secondary property label was meant as a comment or clarification to go with the physical property. For instance, have a look at data set [287](https://materials.hybrid3.duke.edu/materials/dataset/287). The physical property on the x-axis is energy but the axis label is "relative to Fermi level". Alternatively, we could have introduced "energy relative to Fermi level" as a new physical property and left the label empty. In most cases, the physical property name is descriptive enough and the property label should rarely be used. It was really only meant for exceptional cases, where it was important to clarify something about an axis but you didn't want to introduce a new physical property. That said, I personally think it's better to remove the property label field and only use the dropdown menus for selecting the properties (I could make that change if necessary). I've noticed before that the property label fields are confusing to some users. 

The dropdowns for data set and material numbers should be easy to add. I'll get back to you shortly.

**Comment by beccalau** [comment]
Hi @raullaasner !

Another (hopefully quick!) fix would be the atomic structure format. More specifically, as compounds begin to have more and more atomic structures (i.e. I am adding a quite a few to (PEA)2PbI4 right now), you can no longer see the jsmol window/3D visual of the atomic structure. Do you happen to know how we can go about fixing this? Thank you so much for all of your help!!

**Comment by raullaasner** [comment]
I added two select menus to the search page - one for material IDs and one for data set IDs.

It is also now possible to visualize all atomic structures, but only if you go to the page of a given data set. If you try to view all the atomic structures at once ("See all entries for this property"), then the atomic structure is visualized only for the first data set. This is a bug with JSmol that I don't know how to fix at the moment.

Also, I pushed the latest commits to GitHub but somebody still needs to pull them from the server side. I can't ssh into any of Duke's servers at the moment. I opened an issue with OIT and they are looking into it but they don't know how long it takes to fix it.
